docs/dev/devel-env.rst

```diff
@@ -218,8 +218,14 @@ CTRL^C. Once everything is running, visit your new site from your browser!
 Now, you are all set to begin making changes and seeing them in real-time inside
 of your development environment!
 
+Once you are done with your work, shut down the containers using the following command. 
+
+.. code-block:: bash
+
+    ./develop.sh down
```

Comment:
I am not sure if it is required or not, I mean removing the containers everytime you are done with the work is not necessary.

Reply:
I faced the error `datanode is already running` if I don't shut down the containers. I will look into it again.

Reply:
ah! Good enough.

Reply:
So this works for now :)